lib/compliance/parsepdf417-us.js

```diff
@@ -79,7 +93,7 @@ function normalize (result, version, issuingAuthorityNumber) {
   if (result.firstName) result.firstName = result.firstName.split(',')[0]
   var date = result.dateOfBirth.replace(/-/g, '')
   var expirationDate = result.expirationDate ? result.expirationDate.replace(/-/g, '') : null
-  result.country = result.country || 'USA'
+  result.country = result.country || getCountry(result.state)
```

Comment:
Might be better to still leave 'USA' as the default. There might be some problem states out of spec.